src/Workspaces/Core/Portable/EmbeddedLanguages/RegularExpressions/RegexCharClass.cs

```diff
@@ -0,0 +1,350 @@
+﻿using System;
```

Comment:
Should there be a header?

Reply:
so... this is a copy of the code in corefx.  So i would prefer to not edit it much (or at all).

Reply:
@sharwell What's the right thing to do here?

Reply:
I've asked about this and will update you when I hear back.